docs/content/plugins/_index.md-moved-to-intro

```diff
@@ -0,0 +1,34 @@
+---
+title: Plugins
+description: Learn what a Porter plugin can do and see a listing of available plugins
+---
+
+The Porter client is extensible and anyone can write a plugin to integrate with
+Porter. Plugins extend the Porter client, reimplementing Porter's default
+functionality.
```

Comment:
```suggestion
Plugins extend the Porter client, reimplementing Porter's default functionality. The Porter client is extensible and anyone can write a plugin to integrate with Porter.
```

Reply:
I renamed this and forgot to delete this as it was moved to a new directory to be easily identified using the Hugo template.